modules/running-network-verification-manually-cli.adoc

```diff
@@ -7,7 +7,7 @@
 [id="running-network-verification-manually-cli_{context}"]
 = Running the network verification manually using the CLI
 
-You can manually run the network verification checks for an existing {product-title} (ROSA) cluster by using the ROSA CLI (`rosa`).
+You can manually run the network verification checks for an existing {product-title} (ROSA) cluster by using the {product-title} (ROSA) CLI (`rosa`).
```

Comment:
the full product title doesn't need to be expanded twice in the same sentence; the second "{product-title} (ROSA)" can just be "ROSA"